### PR TITLE
New User: LeSalon

### PR DESCRIPTION
**Pull request by nabilfreeman**
We love Babel! 😍

### CODE REVIEW

File: images/users/lesalon.svg [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (xtuc) [inline images/users/lesalon.svg]: Could you please run your SVG through [svgo](https://jakearchibald.github.io/svgomg/) to optimize it ?

**Comment by existentialism** [comment]
@nabilfreeman thanks for the PR! mind resizing the svg to 500x200 and (as @xtuc mentioned, run through svgo)?

**Comment by nabilfreeman** [comment]
@xtuc @existentialism - thanks for your advice. I've made the suggested changes (and merged in the latest commits from master).

Let me know if you need anything else to merge this!

**Comment by existentialism** [comment]
@nabilfreeman thanks!

**Pull request merged**